docker-compose.yml

```diff
@@ -0,0 +1,23 @@
+version: "3.8"
```

Comment:
Can you add a comment that this is only for folks developing atlantis.

Reply:
Sure